rclpy/CMakeLists.txt

```diff
@@ -175,6 +175,7 @@ if(BUILD_TESTING)
       test/test_messages.py
       test/test_node.py
       test/test_parameter.py
+      test/test_parameter_client.py
```

Comment:
just forgot to be included here?

Reply:
https://github.com/ros2/rclpy/pull/959
I find this PR to add test_parameter_client.py but not modify CMakeLists.txt.   
And following PRs never change CMakeLists.txt.  

@ihasdapie Could I ask a question? Why `test_parameter_client.py` isn't added to CMakeLists.txt ?

Reply:
Seems like a mistake in the original PR, adding it here should be fine